addons/mrp/models/stock_move.py

```diff
@@ -290,6 +290,19 @@ def create(self, vals_list):
                     values['location_dest_id'] = mo.location_dest_id.id
         return super().create(vals_list)
 
+    @api.onchange('move_line_ids')
+    def _onchange_move_line_ids(self):
```

Comment:
For me you don't need the hack with hasattr but instead use different methods name

Reply:
Good point, I've fixed it :)